Makefile

```diff
@@ -238,7 +241,7 @@ ISO_ALLOC_PRINTF_SRC = $(SRC_DIR)/iso_alloc_printf.c
 BUILD_DIR = build
 LDFLAGS = -L$(BUILD_DIR) -lisoalloc
 
-all: library tests
+all: testsm
```

Comment:
As in "test secure malloc" or "tests malloc"?

Reply:
As in thats a typo :D